src/main/kotlin/no/nav/helse/prosessering/v1/ettersending/PreprosessertEttersendingV1.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.helse.prosessering.v1.ettersending
+
+import no.nav.helse.aktoer.AktørId
+import no.nav.helse.prosessering.v1.PreprossesertSøker
+import java.net.URI
+import java.time.ZonedDateTime
+
+data class PreprosessertEttersendingV1(
+    val sprak: String?,
+    val soknadId: String,
```

Comment:
Hvorfor er dette soknadId, når EttersendingV1 er søknadId?

Reply:
Det er tilsvarende for den vanlige søknaden, så fulgte bare samme mønster. Kan endres hvis det ikke er gjort av noen god grunn der

Reply:
Hmm ok. Isåfall får vi la det være foreløpig.